### ISSUE TITLE
The Gallery seems to be not available?

### ISSUE THREAD

**Issue by rfcdejong** [OrchardCMS/Orchard.Gallery#36]
From within the dashboard not found at url

http://packages.orchardproject.net/FeedService.svc

and 

https://orchardgallery.azurewebsites.net/api/FeedService is stopped ?

**Comment by urbanit** [comment]
Orchard Gallery is here: [https://gallery.orchardproject.net/api/FeedService](https://gallery.orchardproject.net/api/FeedService)

**Comment by zakgodwin** [comment]
I performed a fresh install of 1.10. but I'm not able to install any modules or themes from the gallery. When I try it explains that the Modules and/or Themes directories do not have correct perms. I've confirmed that they do and I am able to install both Themes & Modules from local source providing the nupkg file. I have the feed url point to https://gallery.orchardproject.net/api/FeedService as provided by @urbanit. The download link doesn't work at all it appears to be pointed to an old package source.
